integration_tests/can_launch/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
Removed `integration_tests` for now and rolled the sample test into `flutter create`